sys-process/runit/runit-2.2.0.ebuild

```diff
@@ -50,22 +48,39 @@ src_install() {
 	dobin $(<../package/commands)
 	dodir /sbin
 	mv "${ED}"/usr/bin/{runit-init,runit,utmpset} "${ED}"/sbin/ || die "dosbin"
-	if use split-usr ; then
-		dosym ../etc/runit/2 /sbin/runsvdir-start
+	if use scripts ; then
+		if use split-usr ; then
+			dosym ../etc/runit/2 /sbin/runsvdir-start
+		else
+			dosym ../../etc/runit/2 /sbin/runsvdir-start
+		fi
 	else
-		dosym ../../etc/runit/2 /sbin/runsvdir-start
+		exeinto /sbin
+		newexe "${FILESDIR}"/2-${ver_runit_cfg} 2
```

Comment:
Is this will install `/sbin/2` when `USE=-scripts`? What's for?

Reply:
typo, should install /sbin/runsvdir-start